src/invitations.ts

```diff
@@ -0,0 +1,204 @@
+import Errors from './errors'
+import {
+  type Response,
+  type Invitation,
+  type InvitationsPage,
+  type QueryParams
+} from './defs'
+
+export default class Invitations {
+  // Invitations API client
+  /**
+   * @class Invitations - Invitations API client
+   * Invitations is used to send, accept, and delete invitations.
+   * @param {string} invitationsUrl - The URL of the invitations service.
+   * @returns {Object} - The Invitations object.
+   */
+  private readonly invitationsUrl: URL
+  private readonly contentType: string
+  private readonly invitationsEndpoint: string
+  private readonly invitationError: Errors
+
+  public constructor (invitationsUrl: string) {
+    this.invitationsUrl = new URL(invitationsUrl)
+    this.contentType = 'application/json'
+    this.invitationsEndpoint = 'invitations'
+    this.invitationError = new Errors()
+  }
+
+  public async SendInvitation (invitation: Invitation, token: string): Promise<Response> {
+    // SendInvitation sends an invitation to the email address associated with the given user.
+    /**
+     * @method SendInvitation - sends an invitation to the email address associated with the given user.
+     * @param {Object} invitation - The invitation object.
+     * @param {string} token - The user's access token.
+     * @returns {Object} - The response object which has a status and a message.
+     * @example
+     * const invitation = {
+     *  userID: '<userID>',
+     *  domainID: '<domainID>',
+     * relation: '<role>' // available options: 'admin', 'owner', 'viewer', 'editor'
+     * }
+     */
+    const options: RequestInit = {
+      method: 'POST',
+      headers: {
+        'Content-Type': this.contentType,
+        Authorization: `Bearer ${token}`
+      },
+      body: JSON.stringify({ user_id: invitation.userID, domain_id: invitation.domainID, relation: invitation.relation })
```

Comment:
Once the issue in defs.ts is solved, you can just use 
```suggestion
      body: JSON.stringify(invitation)
```